lending_pool/src/lib.rs

```diff
@@ -129,17 +172,15 @@ pub trait LendingPool:
     fn liquidate_endpoint(
         &self,
         liquidate_unique_id: BoxedBytes,
-        #[var_args] initial_caller: OptionalArg<Address>,
+        #[var_args] caller: OptionalArg<Address>,
         #[payment_token] asset: TokenIdentifier,
         #[payment_amount] amount: Self::BigUint,
     ) -> SCResult<()> {
```

Comment:
This is more of a personal preference of mine, but I've seen it adopted in many places. The order of arguments should be:
- payment args
- "normal" arguments
- var args

Reply:
will be refactored entirely, but changed the args order.